src/components/LandingPage.jsx

```diff
@@ -1,11 +1,22 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 
 const LandingPage = () => {
+  const [user, setUser] = useState([]);
+
+  useEffect(() => {
+    fetch('/home')
```

Comment:
I suspect that Travis is failing cos of something about the `fetch` method. Could you try to pull in `axios` and then use `axios.get` there?

Just as a note, the response from `axios` that you need is on the data object, which means that `.then` returns `response`, and then `message` will be in `data`.

```.then(response => {
  const { message } = response.data;
  setUser(message);
```